ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2114,6 +2114,19 @@ router_ospf:
         - < interface_1 >
         - < interface_2 >
       max_lsa: < integer >
+      timers:
+        lsa:
+          rx: < 0-600000 - Min interval in msecs between accepting the same LSA >
+          tx:
```

Comment:
Maybe flatten by using `tx_delay`?

Reply:
Flatten `spf` and `tx` to `spf_delay` and `tx_delay` with commit e4265b9